apm-sniffer/apm-sdk-plugin/httpasyncclient-4.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/httpasyncclient/v4/SessionRequestConstructorInterceptor.java

```diff
@@ -31,7 +31,7 @@
 public class SessionRequestConstructorInterceptor implements InstanceConstructorInterceptor {
     @Override
     public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
-        if (ContextManager.isActive()) {
+        if (ContextManager.isActive() && !ContextManager.activeSpan().isExit()) {
```

Comment:
An ES created span is not `ExitSpan`?

Reply:
when ES exit span. this do not propagating context